src/setup.erl

```diff
@@ -134,6 +140,10 @@ enable_cluster_int(Options, no) ->
             config:set("httpd", "bind_address", binary_to_list(NewBindAddress))
     end,
 
+    NodeCount = couch_util:get_value(node_count, Options),
+    ok = require_clustersize(NodeCount),
+    config:set_integer("cluster", "n", NodeCount),
```

Comment:
I'm sure larger values haven't been extensively tested, but IIUC they should work fine. N should be odd, of course.

The parameter exists to allow users/operators to manage the fault tolerance of their system, or a database within it. It should be configurable, and 3 is a very good default - it allows ~unimpeded operation of the system when one copy of the data is unavailable. If a user is willing to pay more latency, CPU, and disk space for additional fault tolerance, they can increase it.